src/m365/flow/commands/run/run-get.ts

```diff
@@ -24,14 +26,12 @@ class FlowRunGetCommand extends AzmgmtCommand {
     return 'Gets information about a specific run of the specified Microsoft Flow';
   }
 
-  public defaultProperties(): string[] | undefined {
-    return ['name', 'startTime', 'endTime', 'status', 'triggerName'];
-  }
-
```

Comment:
TBH I do not think this was a mistake 🤔 we may use default properties also for `get` type commands if one of the returned properties makes the response unreadable or unusable and I think this is actually this case. 
please check the response for text before 
![image](https://user-images.githubusercontent.com/58668583/228386509-38e59ded-dbb0-433e-b08c-a17579323b4e.png)

and now I when it's removed we have
![image](https://user-images.githubusercontent.com/58668583/228386563-0efca029-5087-415f-9750-9dfaa973e5cf.png)

and the same for `csv` response (as it is also considered here)
before
![image](https://user-images.githubusercontent.com/58668583/228386641-897d4259-f82b-46a4-b7ac-732c81c5a8bb.png)

and now
![image](https://user-images.githubusercontent.com/58668583/228386662-0f9cc3b3-dbb7-4669-8dd9-c612b4ba797e.png)

I would revert this change and then revert all other changes that base on it and then lets move forward with the review 😉.

Also if we would modify the default output of a command we should then consider this a breaking change 😉

Reply:
I'm not quite sure about this idea. Personally, I would want all the information I can get from a `get` command. The nested property `properties` should be stripped out indeed for `text`, `csv` and `markdown`. They do cause an overload of information to the terminal. But if I recall correctly, we have separate issues where we are discussing this topic: #4241 #4150 

> Also if we would modify the default output of a command we should then consider this a breaking change 😉

If we add more information to the output and don't remove any of the older properties, I wouldn't see this as a breaking change

Reply:
In JSON output we are returning always all output as if I remember well the default properties are for CSV and text output 🤔. So it is still possible to get all from a get command.

The text output idea was suppose to give a more readable output in the terminal which if we would return properties now is not the case 😉.

You are right on the breaking change👍. If we just add properties then we are safe here 👍. Thanks for pointing it out 👍

Reply:
Hi Adam

Thanks for the review. I'm award that defaultProperties on GET commands are possible, however I contacted my friend Milan about this case and he told me to remove it. I could readd it though, however this wouldn't be breaking I think ass we ADD extra output.

Reply:
@MathijsVerbeeck that's totally fine 👍. You are doing an awesome work 🤩. It's probably me who is making 'problems' here 😅 but since I already started I would like to explain my arguments 
I would not include this change as:
1. first of all it's not part of the related issue and I think even if it's still a small change (for sure in terms of project, but is it small in terms of this command? probably as well 😉) we try to avoid doing more then was specified at that time. 👍 and it's also the case here 😅
2. secondly I think the `text` output mode aim is still to have a readable content that will fit the terminal for the user to have a check. Of course including `properties` still makes it possible for me to read the output on terminal (reading from left to right 😉) but does it look nice. I would say no here 😅
3. I agree that if we add more content to the output as @Jwaegebaert pointed out (and I was wrong 👍) it will not be a breaking change. But if our further work will continue to strip out nested properties from output like `text` and `csv` then this will be a breaking change (at least in this command) as we will then show less. As a result it may happen it will target later releases. From user perspective I would get one release (in the middle) where I get more in the `text` output and then when more our work will be done I will get another release which will give me again less in `text` output 🤔. Or am I missing something here 😉. Maybe we should adjust this when the issues #4241 and #4150 are done 😊

> I would want all the information I can get from a get command

4. as mentioned before it is still possible to get all the output when just using `json` which is default 👍. So it is still possible even now and by default. I don't see why we need to see all in `text` output 😊

@milanholemans could you also add your 3 cents then 😊.
@pnp/cli-for-microsoft-365-maintainers any feedback appreciated 👍

BTW sorry guys for making 'problems' 😅 all in all I hope we will manage to sort it out fast to make this PR ready to be merged 👍 I wish you all an awesome day 🤩

Reply:
> I'm aware that defaultProperties on GET commands are possible, however I contacted my friend Milan about this case and he told me to remove it.

Can't remember this, but it's possible 😊

>     1. first of all it's not part of the related issue and I think even if it's still a small change (for sure in terms of project, but is it small in terms of this command? probably as well 😉) we try to avoid doing more then was specified at that time. 👍 and it's also the case here 😅

Agree, we should stick to the issue as much as possible. However, personally I tolerate small improvements to commands. The question is whether this is a small improvement.

>     2. secondly I think the `text` output mode aim is still to have a readable content that will fit the terminal for the user to have a check. Of course including `properties` still makes it possible for me to read the output on terminal (reading from left to right 😉) but does it look nice. I would say no here 😅

Agree, but if I'm not mistaken, we only use `defaultProperties` for list commands, because if we display too many properties, the command output will become too wide so it's not really readable anymore. For `get` commands we don't have this problem because it only returns 1 object. In this case all properties are printed vertically line by line.
So I think ideally this command shouldn't have `defaultProperties`, the question is that it is the right time to remove them now, or wait for another issue or major version.

Reply:
> Agree, but if I'm not mistaken, we only use `defaultProperties` for list commands, because if we display too many properties, the command output will become too wide so it's not really readable anymore.

@milanholemans but does presetting JSON object in text makes it still readable 
![image](https://user-images.githubusercontent.com/58668583/228530731-dcc3c387-806e-41bd-8a5a-27eb890eb9ce.png)

IMO it does not look nice and that is probably why we have the some same approach in other `get` commands 🤔
 
![image](https://user-images.githubusercontent.com/58668583/228531356-193c459e-d86f-4bf3-bbcc-2fce63ed5eee.png)

I mean what is the point of having JSON and text output if we still preset json in `text` 😅

Reply:
Totally agree @Adam-it that the nested property doesn't look clean in text output but isn't this something for a separate issue? As @milanholemans mentioned, `defaultProperties` is used to control the output table width:

![afbeelding](https://user-images.githubusercontent.com/38426621/228556264-c8948ad3-3fb4-4f2f-8639-e8764543be96.png)

In this scenario, it is logical that we drop some properties so that everything is nice and readable. Maybe it's an idea to create a simular issue like #4241 but for the output type `text`

Reply:
@Adam-it I'm not saying it is clean that a JSON is printed in a text output. Ideally every command should have a text, CSV and MD friendly output, but that's just so much work. Currently we are also printing JSON properties in Markdown and CSV output at some places.

From what I know is that we agreed to only use `defaultProperties` at list commands so you can keep the result clean like Jasey pointed out. In this case, according to me we should just return the entire result when the output is `json`. When the output is `text` or something else, we should do some customizations on the result so it looks clean on text or other outputs. In this case we don't really need `defaultProperties` because we are creating a custom result with only the properties we want to show. `defaultProperties` is used to only show a small portion of the entire result.

Reply:
> In this case, according to me we should just return the entire result when the output is `json`. 
 
 it is already working on this

>  When the output is `text` or something else, we should do some customizations on the result so it looks clean on text or other outputs. We don't really need `defaultProperties` here because we are creating a custom result with only the properties we want to show.

are we really creating such a custom result 🤔 I guess we are returning the `any` response and just adding `triggerInformation`. Anyway since we want to 1. remove defaultProperties and then 2 customize the output I guess we are still half way in this change 😅. I this is something I would do as a separate issue for all `get` commands that have his problem 😊👍

Reply:
> Totally agree @Adam-it that the nested property doesn't look clean in text output but isn't this something for a separate issue?

Indeed it is. If we extend the output now with more data and then with different issue trim to have less data in output then we will have a breaking change 🤔😉. If we leave it as is we may fix it properly when the time is right.

Reply:
> are we really creating such a custom result 🤔 I guess we are returning the `any` response and just adding `triggerInformation`. Anyway since we want to 1. remove defaultProperties and then 2 customize the output I guess we are still half way in this change 😅. I this is something I would do as a separate issue for all `get` commands that have his problem 😊👍

Agree, but that's something that no one likes to do I guess. It's a lot of work.

Reply:
I guess we are going in circles here with comments 😅. 
can we all agree to revert this change and move on or we really want to push this now to have 'a bit of a problem` later with trimming output to have less 😊.
is this JSON in `text` output that important 😅 also since we have it in the default `json?

Reply:
To end the discussion and have the least problems, I'd vote to revert the change.

Reply:
OK for me, but then I have to add the extra field to the defaultProperties I assume?

Reply:
here we go again 😅
@MathijsVerbeeck  you mean additional `defaultProperty` for `triggerInformation`? 
I would say no 😉. as we will still have it in `json` mode and in `text` it will be still JSON in text right 😅 so the same case as we have with `properties` which makes this view not readable/nice

Reply:
I suggest we leave it as it was and forget the hassle. Do a refactor later on when we check which commands need a cleanup.
It's something similar to https://github.com/pnp/cli-microsoft365/issues/4444